docs/tools.md

```diff
@@ -17,17 +16,19 @@ The [MyST Sphinx parser](https://myst-parser.readthedocs.io/en/latest/)
 allows users to write their entire documentation in
 markdown, while utilizing all of the main features of Sphinx.
 
-# MyST-NB
+## MyST-NB
 
 ([link to documentation](https://myst-nb.readthedocs.io/))
 
 MyST-NB allows users to parse Jupyter Notebooks directly into their Sphinx
-documentation, utilizing the MyST Sphinx parser under the hood. It is also
+documentation, utilizing the MyST Sphinx parser under the hood. This allows
+readers to leverage the MyST syntax in order to build *computational
```

Comment:
Perhaps you could add "in markdown cells" after "syntax".

And the second "leverages" in the sentence could be switched to "builds on"